packages/itwinui-css/src/button/classes.scss

```diff
@@ -4,67 +4,54 @@
 
 .iui-button {
   @include iui-button;
-  @include iui-button-size;
-
-  &.iui-active {
-    @include iui-button-active;
-  }
-
-  &.iui-default {
-    @include iui-button-default;
-  }
+}
 
-  &.iui-high-visibility {
-    @include iui-button-high-visibility;
-  }
+.iui-button-label {
+  @include iui-button-label;
+}
 
-  &.iui-cta {
-    @include iui-button-cta;
-  }
+.iui-button-icon {
+  @include iui-button-icon;
+}
 
-  &.iui-idea {
-    @include iui-button-idea;
-  }
+// #region Button variants
+[data-iui-button-variant='default'] {
+  @include iui-button-default;
+}
 
-  &.iui-borderless {
-    @include iui-button-borderless;
-  }
+[data-iui-button-variant='borderless'] {
+  @include iui-button-borderless;
+}
 
-  @include iui-button-size(medium);
+[data-iui-button-variant='high-visibility'] {
```

Comment:
Can you explain your thought process on `data-iui-button-variant` vs `data-iui-variant`? Same for `data-iui-button-sizing`.

Reply:
My thinking was I could make the data attribute more specific and keep the specificity lower:
```scss
.iui-button { … }
[data-iui-button-variant] { … }
```
Versus this which has a higher specificity:
```scss
.iui-button {
  &[data-iui-variant] { … }
}
```